packages/block-library/src/query-pagination/query-pagination-arrow-controls.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import {
+	__experimentalSegmentedControl as SegmentedControl,
+	__experimentalSegmentedControlOption as SegmentedControlOption,
+} from '@wordpress/components';
+
+export function QueryPaginationArrowControls( { value, onChange } ) {
+	return (
+		<SegmentedControl
+			label={ __( 'Arrow' ) }
+			value={ value }
+			onChange={ onChange }
+			help={ __(
```

Comment:
I've added a help text and the `Arrow` label is now visible. Any suggestions for the `copy` here are welcome.